src/cls/rbd/cls_rbd_client.h

```diff
@@ -545,45 +546,162 @@ int mirror_image_snapshot_set_copy_progress(librados::IoCtx *ioctx,
                                             snapid_t snap_id, bool complete,
                                             uint64_t copy_progress);
 
```

Comment:
Line 378 states " operations on the rbd_mirroring object".  Should the mirror group list functions be moved to that section?
Else add a comment stating that these are mirror group operations so it doesn't see like these are part of the image mapping operations (comment on line 519).

Unrelated to this PR but lines 533 to 547 are also not image mapping routines.